docusaurus.config.js

```diff
@@ -84,43 +86,28 @@ const config = {
       footer: {
         logo: {
           alt: 'Logto Logo',
-          src: 'img/footer-logo.svg',
+          src: 'img/silverhand.png',
         },
         style: 'dark',
         links: [
           {
-            title: 'Docs',
-            items: [
-              {
-                label: 'Docs',
-                to: '/',
-              },
-            ],
+            label: 'Docs',
+            to: '/',
+          },
+          {
+            label: 'GitHub',
+            href: 'https://github.com/logto-io/logto',
           },
           {
-            title: 'Community',
-            items: [
-              {
-                label: 'GitHub',
-                href: 'https://github.com/logto-io/logto',
-              },
-            ],
+            label: 'About Us',
+            href: 'https://logto-io.github.io/website/about',
           },
           {
-            title: 'More',
-            items: [
-              {
-                label: 'About Us',
-                href: 'https://logto-io.github.io/website/about',
-              },
-              {
-                label: 'Contact Us',
-                href: 'mailto: contact@logto.io'
-              }
-            ],
+            label: 'Contact Us',
+            href: 'mailto: contact@logto.io',
           },
         ],
-        copyright: `Copyright © ${new Date().getFullYear()} Silverhand Inc. Built with Docusaurus.`,
```

Comment:
would like to keep this since we didn't do the major design work of this site

Reply:
updated